packages/esm-patient-labs-app/src/test-results/panel-timeline/helpers.tsx

```diff
@@ -132,12 +132,12 @@ export const RowStartCell = ({ title, range, units, shadow = false, testUuid, is
     >
       <span className={styles['trendline-link']}>
         {!isString ? (
-          <p
+          <span
```

Comment:
`<p>` is a block-level element whereas `<span>` is not. Nesting it inside a `<span>` would raise a validateDOMNesting warning.

Reply:
Done